### ISSUE TITLE
Most detectors don't seem to return Promise<bool>

### ISSUE THREAD

**Issue by silvanshade** [GoogleChromeLabs/wasm-feature-detect#15]
Just as a little experiment I put together a rust crate at [wasm-feature-detect-sys](https://github.com/interfaces-rs/wasm-feature-detect-sys) so this package is usable in rust projects targeting wasm.

One issue I ran into when testing the bindings is that most of the detectors don't seem to be returning `Promise` as expected. The only one that seems to actually do so is the detector for `big-int`. Looking at the source, this might make sense because that one exported via an `async fn` wrapper.

Should all of the detectors be exported that way or is there maybe something else going on? 

Specifically, what happens when I try to specify the bindings for the other detectors as returning `Promise` is that the test suite throws an error as soon as I try to `await` the result (below).

Everything works if I bind the other detectors as just returning a boolean.

```
JS exception that was thrown:
    TypeError: getObject(...).then is not a function
        at module.exports.__wbg_then_2b35dcc92370b6f9 (/Users/dwm/GitHub/interfaces-rs/wasm-feature-detect-sys/target/wasm32-unknown-unknown/wbg-tmp/wasm-bindgen-test.js:664:37)
        at js_sys::Promise::then2::ha473ddd21e43b7ec (wasm-function[1061]:0x802d8)
        at <wasm_bindgen_futures::JsFuture as core::convert::From<js_sys::Promise>>::from::he6167b32c7c240f3 (wasm-function[110]:0x2c3c1)
        at node::bulk_memory::{{closure}}::he2fecab6491385ce (wasm-function[127]:0x314b4)
        at <std::future::GenFuture<T> as core::future::future::Future>::poll::{{closure}}::h7aa0989309ca8467 (wasm-function[622]:0x67fb4)
        at std::future::set_task_context::hed2d64181cb894d6 (wasm-function[1011]:0x7df97)
        at <std::future::GenFuture<T> as core::future::future::Future>::poll::h517e182ff2787e06 (wasm-function[1372]:0x8b75a)
        at std::future::poll_with_tls_context::{{closure}}::h991891127ed4b723 (wasm-function[1841]:0x980e6)
        at std::future::get_task_context::h63afacc59c480153 (wasm-function[916]:0x79661)
        at std::future::poll_with_tls_context::hc24205f678b61725 (wasm-function[1901]:0x99580)
```

The tests are [here](https://github.com/interfaces-rs/wasm-feature-detect-sys/blob/master/tests/node.rs).

**Comment by surma** [comment]
You are absolutely right. I kinda assumed that everyone would be using this library with `await`, where returning `Promise<T>` or just `T` doesn’t make a difference. I guess I should make sure the return type is consistent for people wanting to use it with the `.then()` notation anyway.

**Comment by surma** [comment]
Published in v1.2.2 ;)

**Comment by silvanshade** [comment]
Thanks! I just updated to the new version and it's working perfectly now 👍
